docs/features/software-templates/input-examples.md

```diff
@@ -0,0 +1,209 @@
+---
+id: input-examples
+title: Built-in examples inputs
+description: Some examples to use in your template
+---
+
+All the examples on this page you can test using _create/edit_ from your Backstage installation.
```

Comment:
Not quite sure what this refers to tbh? You mean just the create page in general? Perhaps refer to the previous documentation on writing templates or something instead? alternatively the template editor.

Reply:
What I mean is that the examples can be copied and tested in the template editor page of the Scaffolder.

As it's a recent feature, I thought it would be a good idea to mention it.